dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -908,6 +908,18 @@ const backgroundMatchNav = (): string => {
 	return '#FFE500';
 };
 
+const cricketScoreboardTop = (): string => {
+	return sport[300];
+};
+
+const cricketScoreboardLink = (): string => {
+	return sport[300];
+};
```

Comment:
Do we need this distinction yet?

Reply:
I think this is an interesting question and has come up a couple of times recently. If two different pieces of UI are nearby, e.g. a link and an adjacent border as in this case, and follow the same colour rules, do you use the same palette function?

I'd be inclined to agree with Olly and say they should be different even if the code looks the same because a) they're semantically different and b) they may diverge in future. But you could also argue that if they diverge you can split them up then, so I'd be interested to hear what people think.

cc @iainjchambers-guardian @MarSavar because the same principle applies to `editorialPalette`.

Reply:
I completely agree that these should be members of different subsets of `palette`, namely `text` and `border`.

I think that you’re seeing it this way: 

```mermaid
graph LR
  decidePalette --> text & border
  text --> cricketScoreboardLink --> sport300
  border --> cricketScoreboardTop --> sport300
```

And I’d be inclined to do:

```mermaid
graph LR
  decidePalette --> text & border
  text --> cricketScoreboardColour
  border --> cricketScoreboardColour
  cricketScoreboardColour --> sport300
```

Reply:
Regardless of of anyones opinions here, I think we can all say with confidence that you went above and beyond with those diagrams 😍😍😍

--- 

I decided to take another look at palette and see what stood out to me in terms of practices for the file.

One thing I noticed was that outside of a few cases, all `palette.text` function were prefixed with `text`, and the same for border, fill, etc.

For example:
```js
{
    text: {
        richLink: textRichLink()
    },
    border: {
         rickLink: borderRickLink()
    }
}
```

I think it makes sense in this case to continue this pattern, so my additions would now look like

```js
{
    text: {
        cricketScoreboardLink: textCricketScoreboardLink()
    },
    border: {
        cricketScoreboardTop: borderCricketScoreboardTop(),
        cricketScoreboardDivider: borderCricketScoreboardDivider()
    }
}
```
While this does result in functions with duplicated responses, it keeps the explicit and ordered pattern which the `decidePalette` file currently uses.

If we wanted to, we could revisit this file (especially as it may be further expanded for fronts?) and see if the patterns used there are still providing the best experience they could!